tests/test_utils.py

```diff
@@ -86,6 +86,116 @@ def test_parse_json_with_comments():
     eq_(result, {'key1': 'value1', 'key2': 'value2', 'key3': 5})
 
 
+def test_parse_json_with_comments_single_line_comments():
+
+    # Need to add comments
+    json_with_comments = ("""{"key1": "value1", //some comments \n"""
+                          """//more comments \n"""
+                          """"key2": "value2", "key3": 5}""")
```

Comment:
I feel like there's a way to make this more readable. Not sure this is it, though.

```suggestion
    json_with_comments = """
    {
        "key1": "value1", //some comments
        //more comments
        "key2": "value2", "key3": 5
    }"""
```